spec/components/post_revisor_spec.rb

```diff
@@ -1117,10 +1123,10 @@
         expect(post.post_uploads.pluck(:upload_id)).to contain_exactly(image1.id, image2.id)
 
         subject.revise!(user, raw: <<~RAW)
-            This is a post with multiple uploads
-            ![image2](#{image2.short_url})
-            ![image3](#{image3.short_url})
-            ![image4](#{image4.short_url})
+          This is a post with multiple uploads
+          ![image2](#{image2.short_url})
+          ![image3](#{image3.short_url})
+          ![image4](#{image4.short_url})
```

Comment:
Let's avoid cosmetic changes here to avoid polluting the commit history.

Reply:
sounds good, this was my editor though, I'll be sure not to include these changes in the future.